README.md

```diff
@@ -38,3 +38,30 @@
 ___
 
 <a href="https://infostart.ru/1c/tools/2169394/" target="_blank"><img alt="Статья на Инфостарт" src="https://infostart.ru/bitrix/templates/sandbox_empty/assets/tpl/abo/img/logo.svg"></a>
+
+
+# OCVC Documentation [English]
```

Comment:
Change - Use level heading `h2`, and delete string `OCVC Documentation`, leave it simple `English`:
## English